lighthouse-core/report/v2/report-renderer.js

```diff
@@ -181,6 +182,36 @@ class ReportRenderer {
   }
 
   /**
+   * @param {string} text
+   * @return {!Element}
```

Comment:
Is the type of element important We're not specifying subclass type for the other methods.